platform/x86/platform.mk

```diff
@@ -21,7 +21,7 @@ LDSCRIPT := $(PLATFORM_DIR)/ldscripts
 
 # Build flags for the device
 CDEFINES :=
-CFLAGS := -Wall -Werror -g -Os -std=c99 -Wno-unused-variable -pedantic \
+CFLAGS := -Wall -Werror -g -Os -std=gnu99 -Wno-unused-variable -pedantic \
```

Comment:
Sigaction requires `#include <signal.h>` and  that include requires the std to be changed. I honestly don't see a problem with this since we are all building on x86 Linux anyway and there should be no impact on anyone.